### DISCUSSION TITLE
No data after pull

### DISCUSSION THREAD

**Discussion by spoutn1k** [firefly-iii/firefly-iii#9173] [Firefly III]
Hello all,

I restarted my firefly instance after a few months and all the data is gone. I am using docker with a docker-compose script. Is this intended ?

**Comment by JC5** [comment]
That depends entirely on the exact commands you've used.

**Reply by spoutn1k** [reply]
`docker compose up`. Unfortunately I did not do anything special, and it seems it pulled the importer's new version, then ran a bunch of migrations, and that resulted in a server that asked me to register.
[out.log](https://github.com/user-attachments/files/16740933/out.log)

**Reply by JC5** [reply]
I'm sorry, I see nothing out the ordinary.

Did you run docker compose down? Because that removes volumes.
